src/app/templates/scripts/_version.js

```diff
@@ -0,0 +1,68 @@
+var exec = require('child_process').exec;
+var fs = require('fs');
+var path = require('path');
+var pkg = require(path.join(__dirname, '../package.json'));
+
+/**
+ * Determines whether or not the project has the CHANGELOG setup by checking
+ * for the presence of a CHANGELOG.md file and the necessary dependency and
+ * npm script.
+ *
+ * @return {Boolean}
+ */
+var hasChangelog = function() {
+  try {
+    fs.statSync(path.join(__dirname, '../CHANGELOG.md'));
+  } catch (x) {
+    return false;
+  }
+  return pkg.devDependencies.hasOwnProperty('chg') &&
+    pkg.scripts.hasOwnProperty('change');
+};
+
+/**
+ * Determines whether or not the project has the Bower setup by checking for
+ * the presence of a bower.json file.
+ *
+ * @return {Boolean}
+ */
+var hasBower = function() {
+  try {
+    fs.statSync(path.join(__dirname, '../bower.json'));
+    return true;
+  } catch (x) {
+    return false;
+  }
+};
+
+var commands = [];
+
+// If the project has a CHANGELOG, update it for the new release.
+if (hasChangelog()) {
+  commands = commands.concat([
+    'chg release "' + pkg.version + '"',
```

Comment:
Yeah. This is run as the npm `"version"` script, at which point the `package.json` has been version-bumped, but it has not been committed yet.